async_producer.go

```diff
@@ -265,10 +265,13 @@ func (p *asyncProducer) topicDispatcher() {
 func (p *asyncProducer) partitionDispatcher(topic string, input chan *ProducerMessage) {
 	handlers := make(map[int32]chan *ProducerMessage)
 	partitioner := p.conf.Producer.Partitioner(topic)
+	breaker := breaker.New(3, 1, 10*time.Second)
```

Comment:
"a bunch" is 2, and they currently have the same settings. We might want to expose those parameters, I'm not sure if anybody really actually cares to tweak them.